modules/snooker/src/main/ts/ephox/snooker/lookup/Blocks.ts

```diff
@@ -59,4 +53,7 @@ const rows = (warehouse: Warehouse): Optional<SugarElement>[] => {
 
 };
 
-export { columns, rows };
+const cols = (warehouse: Warehouse): Optional<SugarElement<HTMLTableColElement>>[] =>
+  Arr.map(Warehouse.justColumns(warehouse), (column: Column) => Optional.from(column.element));
+
+export { cols, columns, rows };
```

Comment:
Could we rename them something less confusing than `columns` and `shortening of the word columns` ?

Reply:
I suspect that will happen down the track; probably will leave it as is for now. Snooker generally needs more consistent terminology.

Reply:
I'd like it to be a bit less consistent than variations of the same word.
And wouldn't it be easiest to rename it now, while we remember what it is?

Reply:
We've discussed it a couple times I think and didn't come up with anything super helpful. Also, deadlines.

Reply:
Esh, deadlines.

Reply:
We removed this and moved it to `ColumnSizes` as that's the only place that needed it and should help to avoid confusion.